python_bindings/stub/PyStubImpl.cpp

```diff
@@ -63,6 +65,23 @@ void install_error_handlers(py::module &m) {
     handlers.custom_error = halide_python_error;
     handlers.custom_print = halide_python_print;
     Halide::Internal::JITSharedRuntime::set_default_handlers(handlers);
+
+    static py::object halide_error = py::module_::import("halide").attr("HalideError");
+    if (halide_error.is(py::none())) {
+        throw std::runtime_error("Could not find halide.HalideError");
+    }
+
+    // Make a subclass of HalideError for this stub to throw.
```

Comment:
Why? Is it even possible to catch this exception specifically in user code if it's dynamically generated?

Reply:
Nah, it's mainly because I can't figure out how to get PyBind to actually get and throw an instance of the base class. I'll revisit this and try to actually figure in out...

Reply:
Looks good now, thanks!